### DISCUSSION TITLE
Disable culling?

### DISCUSSION THREAD

**Discussion by ArdFlamingo** [Bodmer/TFT_eSPI#2784] [Q&A - General]
Is it possible to disable culling so that pixels are still "drawn" outside of the display (or viewport), or at least are drawn again when they come back into the viewport?

Currently, when using the scroll function, if text goes out of the viewport, those pixels get erased forever.

**Comment by Bodmer** [comment]
No, the viewport clipping is inherent in the functioning of the feature.

Note that you can draw into a sprite then render part of that sprite onto the screen so other pixels in the sprite will stil be present (if the sprtie is big enough).
